cmd/root.go

```diff
@@ -251,6 +251,14 @@ func initFlags() {
 		"",
 		"Address of pushgateway e.g. http://localhost:9091",
 	)
+
+	rootCmd.Flags().BoolVarP(
+		flags.ErrorStatusOnFail,
+		"error-status-on-fail",
+		"",
+		true,
+		"If set to false, returns a non-zero exit status",
```

Comment:
Go bool flags are not set they are either present or not so perhaps "Force zero exit status when report errors are present"